volatility3/framework/plugins/linux/tracing/tracepoints.py

```diff
@@ -183,16 +190,21 @@ def parse_tracepoint(
                     probe_handler_address
                 )
             else:
-                vollog.warning(
+                vollog.debug(
                     f"Could not determine tracepoint@{tracepoint.vol.offset:#x} probe handler {probe_handler_address:#x} module origin.",
                 )
 
+            if hasattr(tracepoint_func, "prio"):
+                prio = tracepoint_func.prio
+            else:
+                prio = renderers.NotAvailableValue()
+
             yield ParsedTracepointFunc(
-                utility.pointer_to_string(tracepoint.name, count=512),
+                tracepoint_name,
                 tracepoint.vol.offset,
                 probe_handler_symbol,
                 probe_handler_address,
-                tracepoint_func.prio,
+                prio,
```

Comment:
Should this be `None`, and then `NotAvailableValue` by used just before rendering (only issue would be if prio could be 0, although `prio if prio is not None else NotAvailableValue()` would work)?  APIs should generally work with standard python options if possible...

Reply:
good catch